cluster-autoscaler/expander/expander.go

```diff
@@ -37,6 +37,8 @@ var (
 	// PreferSpotExpanderName selects a random node group with spot instances if possible, or any random node group
 	// otherwise
 	PreferSpotExpanderName = "prefer-spot"
+	// PriorityExpanderName selects the node group with the highest priority
+	PriorityExpanderName = "priority"
```

Comment:
Might conflict with the upstream priority expander in the future, do we want to rename now?